src/main/java/core/basesyntax/model/FruitRecord.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitRecord {
+    private String fruitName;
```

Comment:
better to create Fruit class where will be stored fruit name.
 If we will have more fruit fields in future it will be better to have Fruit class.

Reply:
Fixed